app/models/invenio_rdm_record_converter.rb

```diff
@@ -508,4 +517,18 @@ def funding(file_id)
     # remove nil values
     funding_sources.compact
   end
+
+  def map_collections_to_communities
+    @dh_collections.map do |collection_path|
+      prism_community_id_from_collection_path(collection_path)
```

Comment:
This should be fine. I don't recall if we have some records that were part of a dh collection that are transferred to Prism but are not in a community there. We'll see - might filter them at import if that's the case.